odoo/orm/fields_textual.py

```diff
@@ -707,12 +712,16 @@ def get(self, key, default=None):
             return default
         if vals is None:
             return None
+        if not (self._field.compute or (self._field.store and (key or key.origin))):
```

Comment:
the condition is equavalent to 
```
(not self._field.compute and not self._field.store) or
(not (self._field.compute or key or key.origin)) 
```

I think we don't have  `(not self._field.compute and not self._field.store)` translated field?

Reply:
Let's keep the condition clear and robust, for unexpected cases (database-stored computed fields).